README.md

```diff
@@ -116,24 +154,40 @@ You can set multiple options in your options-object for the output of the slider
 For styling you have two options:
 
 ### 1. Predefined stylesheets
-If you want it simple you can use one of our predefined stylesheets. Right now these stylesheets are availible:
-> TBD. This will come soon. 
-<!-- TODO: Add names, descriptions and images -->
-<!-- - Sheet 1: One Sentence about it + Image
-- Sheet 2: One Sentence about it + Image -->
-
-Just import one into your HTML-document with the following link-tag: 
-<!-- TODO: add link! -->
+If you want it simple you can use one of our predefined stylesheets. Right now this stylesheet is availible:
+- `Minimal`: This is the most basic styles you can use. Pretty clean in Black and white.
+- More are coming soon! We are working on sweet colourful styles.
+
+Just import one into your HTML-document with the following link-tag..
+#### ...when you used a packagemanager:
+```html
```

Comment:
bundling them is also possible via `import "openseadragon-opacity-slider/styles/minimal.css";`